client/common/sass/components/_search-bar.sass

```diff
@@ -0,0 +1,9 @@
+.search-form
+	position: relative
+
+.search-button
+	position: absolute
+	right: 4px
+	top: 5px
+	border: 0
+	font-weight: 700
```

Comment:
Based on the designs, the font-weight should be 500. We are still falling back to Helvetica instead of Helvetica Nueue, so the font-weight 500 looks different in local implementation compared to the design given, but I think sticking with 500 will make it look more proper once we have the correct font.

Reply:
That's a good point! I didn't consider a weight between 400 and 700 but it didn't really look right at either. This seems to work better, for sure.